lib/powerpoint.js

```diff
@@ -167,20 +277,17 @@ class Powerpoint extends EventEmitter2 {
   _createSlides (imageFiles, pptFile, done) {
     var start = this.nowInMillis()
     var pptx = officegen('pptx')
+    var d = this.options.dimensions
+    // https://github.com/Ziv-Barber/officegen/issues/112
```

Comment:
I will do that over in scoreboard-export, want to keep SB concerns out of this module since it is open source.